libraries/Microsoft.Bot.Connector/Authentication/ConstantHttpClientFactory.cs

```diff
@@ -10,7 +10,7 @@ namespace Microsoft.Bot.Connector.Authentication
     /// <summary>
     /// HttpClientFactory that always returns the same HttpClient instance for ADAL AcquireTokenAsync calls.
     /// </summary>
-    internal class ConstantHttpClientFactory : IHttpClientFactory
+    internal class ConstantHttpClientFactory : Microsoft.IdentityModel.Clients.ActiveDirectory.IHttpClientFactory
```

Comment:
Why not add the namespace in a `using` at the top of the file?

Reply:
I kind-a wanted this to be super explicit because there are two HttpClientFactories in play now - this old thing which must have been used before and then the standard one from Microsoft.Extensions.Http